.github/workflows/docker-build-push-dockerhub.yml

```diff
@@ -112,6 +114,43 @@ jobs:
       #   run: |
       #     DOCKER_BUILDKIT=0 docker compose build --build-arg DATABASE_URL=postgresql://${{ env.POSTGRES_USER }}:${{ env.POSTGRES_PASSWORD }}@${{ env.DATABASE_HOST }}/${{ env.POSTGRES_DB }} calcom
 
+      - name: Test runtime
+        run: |
+            docker run --rm --network stack \
+              -p 3000:3000 \
+              -e DATABASE_URL=postgresql://${{ env.POSTGRES_USER }}:${{ env.POSTGRES_PASSWORD }}@database/${{ env.POSTGRES_DB }} \
+              -e NEXTAUTH_SECRET=${{ env.NEXTAUTH_SECRET }} \
+              -e CALENDSO_ENCRYPTION_KEY=${{ env.CALENDSO_ENCRYPTION_KEY }} \
+              docker.io/calcom/cal.com:add-runtime-check &
+              
+            server_pid=$!
+
+
+            echo "Waiting for the server to start..."
+            sleep 120
```

Comment:
We give 120 seconds for the server to be ready to accept curl request. It was erroring out with code 7 if we requested when it wasn't ready.